packages/ui/src/signals/labelClip.ts

```diff
@@ -0,0 +1,11 @@
+import {signal} from '@preact/signals';
+
+interface LabelClipDrag {
+  left: number;
+  dragging: boolean;
+}
```

Comment:
I think this could be expressed with `number | null` instead of this interface.
Right now `left` will be invalid/unusable when `dragging` is false, but there's nothing in the type that implies it.
A nullable number clearly states "if I'm `null` there's no valid number to use"